src/ucs/sys/compiler_def.h

```diff
@@ -181,4 +181,16 @@
 /* Check if an expression is a compile-time constant */
 #define ucs_is_constant(expr)      __builtin_constant_p(expr)
 
+/*
+ * Define code which runs at global constructor phase
+ */
+#define UCS_STATIC_INIT \
+    static void UCS_F_CTOR UCS_PP_APPEND_UNIQUE_ID(ucs_initializer_ctor)()
+
+/*
+ * Define code which runs at global destructor phase
+ */
+#define UCS_STATIC_CLEANUP \
+    static void UCS_F_DTOR UCS_PP_APPEND_UNIQUE_ID(ucs_initializer_dtor)()
+
```

Comment:
why needed?

Reply:
Needed for UCS_CONFIG_REGISTER_TABLE_ENTRY to work as we want to reuse parser.h in xccl.

Reply:
ok